config/routes.rb

```diff
@@ -4,14 +4,7 @@
   # We remove the sign_up path name so as not to allow users to sign in with username and password.
   devise_for :users, :controllers => { :omniauth_callbacks => "omniauth_callbacks" }, path_names: { sign_up: ''}
 
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
-
-  # You can have the root of your site routed with "root"
-  root to: redirect('https://sumofus.org'), as: :sumofus
-
-  # Specifies routing to templates controller for when a new template layout is requested by 
-  # a user toggling different templates when creating a campaign page
+  root 'home#index'
 
```

Comment:
> I had assumed that champaign would be occupying its own subdomain ( eg admin.sumofus.org). If this is going to be the case, then I don't understand why we'd want to redirect users to sumofus.org.

In the long run, that's true, but at the moment, there isn't anything set up as a basic landing page, and we're planning on doing testing before we've got a landing page set up. We don't want users going to the root URL and seeing what's up there now, but we also don't have specs for anything to replace it. The result was that a redirect to `sumofus.org` made the most sense for the time being. Do you have thoughts on an alternative?